src/App.css

```diff
@@ -3091,3 +3092,105 @@ code, .code {
   transform-origin: 0 0; /* see note above */
 }
 .ripple_loader div { box-sizing: content-box; }
+
+.advance-setting-wrapper {
+  display: flex;
+  justify-content: center;
+  margin-bottom: 20px;
+}
+
+p.advance-setting-label {
+  margin-bottom: 5px;
+  line-height: 1.2;
+}
+
+p.advance-setting-description {
+  font-size: 90%;
+}
+
+.advance-setting-link {
+  display: flex;
+  position: relative;
+  transition: opacity 100ms ease-in-out;
+}
+
+a.advance-setting-link { color: #000; }
+.dark a.advance-setting-link { color: #fff; }
+
+.advance-setting-chevron {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  position: relative;
+}
+
+.advance-setting-section {
+  display: flex;
+  justify-content: center;
+  margin: 0 auto;
+  max-width: 34em;
+}
+
+.chevron-up {
+  transform: rotate(180deg);
+}
+
+/* Custom checkbox style starts */
+.checkbox-container {
+  display: block;
+  position: relative;
+  padding-left: 35px;
+  margin-bottom: 12px;
+  cursor: pointer;
+  user-select: none;
```

Comment:
The build process will automatically add "-moz-", "-webkit-" prefixes based on our `browserslist`.

Reply:
Removed